providers/src/airflow/providers/openlineage/CHANGELOG.rst

```diff
@@ -26,6 +26,26 @@
 Changelog
 ---------
 
+1.14.0
+......
+
+Features
+~~~~~~~~
+
+* ``Add support for semicolon stripping to DbApiHook, PrestoHook, and TrinoHook (#41916)``
```

Comment:
Here it's marked as feature

Reply:
The PR added code to openlineage integration which is crucial for the feature so I considered this a feature